client/json-rpc/src/async_client/client.rs

```diff
@@ -362,9 +400,27 @@ impl<R: RetryStrategy> Client<R> {
 
     pub async fn send_without_retry(&self, request: &Request) -> Result<JsonRpcResponse, Error> {
         let id = 1;
-        let rpc_resp: JsonRpcResponse = self.send_json_request(request.to_json(id)).await?;
-        self.validate(&rpc_resp, id, id)?;
-        Ok(rpc_resp)
+        let rpc_resp: Vec<JsonRpcResponse> =
```

Comment:
the following specific logic should be inside your ClusterRequestStrategy, this way, you don't need change batch_send_without_retry method